src/commands/properties/verify.ts

```diff
@@ -0,0 +1,101 @@
+import { BaseCommand } from "../../baseCommand";
+import { Formater, OUTPUT_FORMAT } from "../../formater";
+import { getTargetArg } from "../../utils";
+import { flags as oFlags } from "@oclif/command";
+import { NftFactoryServicesList, VERIFIED_URL_KEY_PREFIX, VERIFIED_URL_JWT, JwtUtils } from "@uns/ts-sdk";
+import { readFileSync } from "fs";
+import { PropertiesUpdateCommand } from "../../updatePropertiesCommand";
+import { PropertyRegisterCommand } from "./register";
+
+export class PropertyVerifyCommand extends PropertiesUpdateCommand {
+    public static description = "Conclud ownership verification processus.";
+
+    public static usage =
+        "properties:verify TARGET --type {url} --url-channel {html,file,whitelist} --url-name {label}";
```

Comment:
As suggested by you :smile: 
```suggestion
        "properties:verify TARGET --type {url} --url-channel {html,file} --url-name {label}";
```